core/sa-edu-dental-fillings.tab.info

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "sa-edu-dental-fillings",
+    "title": "SA: Dental Fillings Dataset",
+    "description": "Toy survival dataset containing data from 20 people regarding how long their dental fillings stayed in place over an observation window of 10 years. It includes four features: Time, Event (dental filling falls out = 1, censoring = 0), Type of material (composite, ceramic), Brushing time (time spent on dental hygiene daily).",
+    "references": [],
+    "tags": [
+        "survival analysis",
+        "censoring"
+    ],
+    "target": "",
+    "version": "1.2",
+    "year": 2022,
+    "instances": 20,
+    "missing": false,
+    "variables": 4,
+    "source": "",
```

Comment:
If you did not generate the file yourself, try to put the link to where the data came from in source or a paper/website citation as a reference.

Reply:
I generated it myself.

Reply:
In that case I suggest you leave this blank, and after you publish a blog using this data you edit this info and add a link to the blog to "seealso" :)